### DISCUSSION TITLE
可以认识一下吗？

### DISCUSSION THREAD

**Discussion by later-3** [QQxiaoming/quard_star_tutorial#10] [General]
可以加个微信吗？ 我在工作中也开始使用qemu，我不会问一些显而易见或者动动手就能在网上搜到的问题，我觉得你做这一件事很酷。

**Comment by later-3** [comment]
我可以帮助一起完善、更新文档

**Comment by later-3** [comment]
我会从头到尾 跟着博客走一遍

**Comment by QQxiaoming** [comment]
好的。因为我没有在你的主页找到你的邮箱，因此你可以提供你的邮箱，以便我将我的微信号发送给你，或者你可以向我的发送一封邮件。如果我们成功通过邮件联系，麻烦关闭这个issue，也许使用discussions更适合这个问题。

**Comment by later-3** [comment]
好滴
